services/queue/src/workclaimer.js

```diff
@@ -151,21 +38,29 @@ class WorkClaimer extends events.EventEmitter {
     this._hintPollers = {}; // provisionerId/workerType -> HintPoller
   }
 
+  getHintPoller(taskQueueId) {
+    if (!this._hintPollers[taskQueueId]) {
+      this._hintPollers[taskQueueId] = new HintPoller(taskQueueId, {
+        monitor: this._monitor,
+        pollPendingQueue: async (count) => this._queueService.pollPendingQueue(taskQueueId)(count),
```

Comment:
`HintPoller` has single dependency `monitor` now. 

It might be that we don't even need this class anymore (at least in its current form). It will likely be rewritten in upcoming PRs to use `listen/notify`